extensions/amp-carousel/0.1/slidescroll.js

```diff
@@ -479,4 +476,19 @@ export class AmpSlideScroll extends BaseSlides {
       this.slidesContainer_./*OK*/scrollLeft = interpolate(pos);
     }, 80, 'ease-out').thenAlways();
   }
+
+  /**
+   * Cancels the touchmove events for the element so that viewer does not
+   * consider the swipes in the carousel as swipes for changing AMP documents.
+   * @private
+   */
+  cancelTouchEvents_() {
+    // TODO(aghassemi): https://github.com/ampproject/amphtml/issues/4742
+    // prevents us from using SwipeXRecognizer with an empty handler to
+    // cancel the events. This work around is not great and temporary until
+    // #4742 is fixed.
+    this.element.addEventListener('touchmove', evt => {
+      evt.stopPropagation();
+    });
```

Comment:
The issue why SwipeX has a problem is likely due to the fact that it was build around `preventDefault`, where this code would kill scrolling on carousel. But since we are no longer doing `preventDefault` here, our issue now is a lot simpler - we can indeed stop any touchmove from propagating. So, I think, it was my wrong advise to use SwipeX that got us into this problem. Not sure if we ever run into a case where, e.g. vertical swipes, should NOT be stopped. But for now, I think this is fine.

Could you please also do this change in `scrollable-carousel.js` in this PR?